src/main/java/com/internet/shop/model/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.internet.shop.model;
+
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
+        this.userId = userId;
+    }
```

Comment:
```suggestion
    public Order(Long userId) {
        products = new ArrayList();
        this.userId = userId;
    }
```
same in ShoppingCart

Reply:
Done